lib/opencensus/stats/aggregation_data/distribution.rb

```diff
@@ -50,14 +53,16 @@ def initialize buckets
           @mean = 0
           @sum_of_squared_deviation = 0
           @bucket_counts = Array.new(buckets.length + 1, 0)
+          @exemplars = []
```

Comment:
The type is hash; should this be `{}` instead of `[]`?

Reply:
Made the change. `exemplars ` is an array type.